assets/training/model_management/src/azureml/model/mgmt/downloader/downloader.py

```diff
@@ -99,6 +99,8 @@ class GITDownloader:
     """Downloader class for model hosted on public git repositories."""
 
     URI_TYPE = PathType.GIT.value
+    FILES_TO_REMOVE = [".gitattributes"]
+    DIRS_TO_REMOVE = [".git"]
```

Comment:
I believe there is an amlignore or something similar as well isn't it?

Reply:
.. not specifically in git downloader but potentially when we are uploading/registering the model.

Reply:
added